### PR TITLE
CDK: Review concepts

### PR DESCRIPTION
**Pull request by kmurphypolygon**
- Updates after review session with team.
- This one reviews the concepts section which reads well, although there is one question about whether some of these docs should be in the architecture section rather than the concept section as they are less generalized.
- Apart from that, these changes are mostly links, and some styling.
- Found the descriptions of rollup, validium, zkEVM etc, compelling.

### CODE REVIEW

File: docs/cdk/concepts/layer2s.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ Under the hood, L2s create "batches" of transactions, and periodically submit ma
...
 
-![L2 Batching Overview](../../img/cdk/l2-overview-diagram.svg)
+![L2 batching overview](../../img/cdk/l2-overview-diagram.svg)
 
 ## Types of layer 2s
 
 L2s come in different shapes and sizes in terms of their relationship with Ethereum: Each design decision comes with trade-offs in terms of security, scalability, or decentralization.
 
-For example, some L2s, such as the [Polygon zkEVM](https://docs.polygon.technology/zkEVM/) send all transaction data to Ethereum, whereas other L2s only send information about the state differences, or choose not to send transaction data to Ethereum; instead relying on different data availability mechanisms.
+For example, some L2s, such as the [Polygon zkEVM](../../zkEVM/) send all transaction data to Ethereum, whereas other L2s only send information about the state differences, or choose not to send transaction data to Ethereum; instead relying on different data availability mechanisms.
```
Comment (EmpieichO) [inline docs/cdk/concepts/layer2s.md]: This link renders a 404. A specific .md file needs to be linked.

**Review by EmpieichO — APPROVED** [review]

**Review by gsilva5 — COMMENTED** [review]
lgtm

**Review by fukinbubba — COMMENTED** [review]
checked

**Review by fukinbubba — COMMENTED** [review]
1

**Pull request merged**